src/screen.c

```diff
@@ -394,8 +403,11 @@ void draw_drag() {
     drawBar(52, 55, 8);
     drawBar(107, 14, 4);
 
-    print4(108, 5, 1, "D5");
-    print(112, 40, 6, UF2_VERSION_BASE);
+    // Center PRODUCT_NAME and UF2_VERSION_BASE.
```

Comment:
The idea with "D5" in the corner was that the user could select the generic board on the board selection page (clicking on D5 or on Pybadge has exactly the same effect; for this reason the instructions also now always say to use `ARCADE-D5` drive). I originally had the generic boards at the top but got overruled. I think as we get more and more boards on that page the generic ones will go back to the top, or the whole thing will be redesigned.

There is space on the left of "D5" for your branding. You can have icon and text there.

The way this is done in STM32 bootloader is like this: https://github.com/mmoskal/uf2-stm32f/blob/master/boards/brainpad_arcade/board.h#L84

I can add similar hooks, or you can do it.

I'm not sure what's the usefulness of saying "pygamer" or "pybadge" on the screen when it already says that on the silk. OTOH it doesn't say you can select D5 from Arcade download menu.

<img width="986" alt="Screen Shot 2019-05-20 at 16 07 18" src="https://user-images.githubusercontent.com/10673976/58057401-907caf00-7b19-11e9-8d0b-5835f36d9596.png">

The brainpad logo was done here: https://makecode.com/_Aa3T8r51e2u3 - when you run the script, it generates the encoding of the image on the console

<img width="348" alt="Screen Shot 2019-05-20 at 16 18 20" src="https://user-images.githubusercontent.com/10673976/58057750-eb62d600-7b1a-11e9-9efa-50a0de09c4e8.png">


cc @ladyada

Reply:
we can replace with other stuff, this was very "lets get it working" - should we make an adafruit logo?

Reply:
Most definitely create a logo! You can use the script I posted to play with it (the logo will need to be in Arcade so it can be encoded).

Reply:
OK, now I see what was going on with the `print4` text writing. I thought I was fixing some casual centering code, but didn't know about the STM32 logos and bootloaders. I see that the BrainPad and the Meowbit both use "ARCADE-F4", and it's hardwired in their bootloaders.

Since our boards are multi-purpose, calling them "ARCADE-something" makes for some confusion for other uses. Would it be possible to get the boot drive name from the config data or somewhere like that? Then could this general message be made more specific when you download something? The written doc would also have to be made a little more general. 

![image](https://user-images.githubusercontent.com/2847802/58062816-dcdfe300-7b49-11e9-97f2-d16205828015.png)

I do want to get our sources in sync as much as possible.

Reply:
I edited the comment above after realizing parts were wrong.

The "Download complete..." message could be changed to "Move the .UF2 file to the ARCADE or BOOT drive...". That would cover the `...BOOT` drives we have. We'll work on customizing the banner/logos for our boards.